tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/InternalReportScenario.kt

```diff
@@ -0,0 +1,17 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import com.bugsnag.android.*
+
+internal class InternalReportScenario(config: Configuration,
+                                      context: Context) : Scenario(config, context) {
+    init {
+        config.setAutoCaptureSessions(false)
+    }
+
+    override fun run() {
+        super.run()
+        sendInternalReport(RuntimeException("Whoops"), config, Bugsnag.getClient())
```

Comment:
Rather than literally calling the internal method for sending an internal report, a good test would create a malformed file in the cache dir and see what happens when it is flushed through the system. This current implementation shows that `reportInternalBugsnagError()` works correctly when called directly but not what would happen if a report file was broken.